src/editor/object_settings.cpp

```diff
@@ -293,7 +296,35 @@ ObjectSettings::add_music(const std::string& text, std::string* value_ptr,
                           std::optional<std::string> default_value,
                           unsigned int flags)
 {
-  add_file(text, value_ptr, key, std::move(default_value), {".music"}, {"/music"}, false, flags);
+  auto generate_help_text_for_file = [](std::string path) -> std::string
+  {
+    std::string filename = FileSystem::basename(path);
+    std::unique_ptr<SoundFile> sound_file;
+    try {
+      sound_file = load_sound_file(path);
+    } catch (...) {
+      return "";
+    }
+
+    std::string& author = sound_file->m_author;
+    std::string& license = sound_file->m_license;
+    std::string& title = sound_file->m_title;
```

Comment:
`const std::string&`

Reply:
Thanks, I pushed a commit that changes this for all local vars in this pull request.